apps/mark-scan/frontend/src/app_root.tsx

```diff
@@ -525,10 +529,14 @@ export function AppRoot({ reload }: Props): JSX.Element | null {
     if (pollsState === 'polls_open') {
       if (
         isCardlessVoterAuth(authStatus) &&
-        // accepting_paper expects poll worker auth. If the frontend sees accepting_paper but has cardless voter auth,
-        // it means the state hasn't caught up to auth changes. We check that edge case here to avoid flicker ie.
-        // rendering the ballot briefly before rendering the correct "Insert Card" screen
-        stateMachineState !== 'accepting_paper'
+        // Handle states that expect poll worker auth. If the frontend sees one of these states but has cardless voter auth,
+        // it means the state hasn't caught up to auth changes. We check that edge case here to avoid flicker eg.
+        // rendering the ballot briefly before rendering the correct "Insert Card" screen.
+        // This problem is caused by conditioning on auth in both the frontend and backend. The long term fix is to
+        // move auth entirely to the backend.
+        // https://github.com/votingworks/vxsuite/issues/3985
+        stateMachineState !== 'accepting_paper' &&
+        stateMachineState !== 'not_accepting_paper'
```

Comment:
Now that we've seen this is a consistent pattern for any state that changes auth in the backend, IMO we should bump the priority of #3985 ahead of adding any additional states with auth changes @arsalansufi @jonahkagan @mattroe

Reply:
That makes sense! I brought the issue up to the medium pri section on the VxMarkScan board. (I don't have a great sense of which issues would require "adding any additional states with auth changes")